avocado/core/runner.py

```diff
@@ -239,7 +239,8 @@ def finish(self, proc, started, timeout, step):
                        test_state['name'])
         if proc.is_alive():
             TEST_LOG.warning("Killing hanged test process %s" % proc.pid)
-            for _ in xrange(5):     # I really want to destroy it
+            end_time = time.time() + 60
+            while time.time() < end_time:
```

Comment:
Ok, that (%10) will keep the number of calls around the same so we only touch the timeout here, which is the PR intention. Sounds good.